src/FixedBasic/Base.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Jitendra\PhpValve\FixedBasic;
```

Comment:
rename `FixedBasic` to `FixedWindow` ?

Reply:
There can be more than one algorithm which has "fixed window" trait.